packages/rke2-calico/generated-changes/patch/templates/crs/custom-resources.yaml.patch

```diff
@@ -16,11 +16,14 @@
 +*/}}
 +{{ if not (empty .Values.global.clusterCIDRv6) }}
 +{{ $myIP6Dict := dict "natOutgoing" "Enabled" "cidr" .Values.global.clusterCIDRv6 }}
++{{ $finalIpPoolList := list $myIP6Dict }}
++{{ if not (empty .Values.global.clusterCIDRv4) }}
 +{{ $allIpPools := get .Values.installation.calicoNetwork "ipPools" }}
 +{{ range $allIpPools }}
 +{{ $_ := set . "encapsulation" "None" }}
 +{{ end }}
 +{{ $finalIpPoolList := append $allIpPools $myIP6Dict }}
++{ end }}
 +{{ $calicoNetwork := get .Values.installation "calicoNetwork" }}
 +{{ $_ := set $calicoNetwork "ipPools" $finalIpPoolList }}
```

Comment:
If `.Values.global.clusterCIDRv6` is empty, finalIpPoolList does not exist, right? Doesn't helm complain about it?

Reply:
It will not go on that if

Reply:
you are right. I'm stupid